plop-templates/Component/Component.tsx.hbs

```diff
@@ -18,13 +21,15 @@ export interface Props {
  */
 export const {{pascalCase name}} = ({
   className,
+  // Add other deferenced props to use
```

Comment:
Make sure the plopped file generates code that isn't immediately invalid to prettier but also a few more instruction areas